packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -281,14 +278,6 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this._attachState = this.containerRuntime.attachState !== AttachState.Detached && this.existing ?
             this.containerRuntime.attachState : AttachState.Detached;
 
-        this.bindToContext = () => {
-            assert(this.bindState === BindState.NotBound, 0x13b /* "datastore context is already in bound state" */);
```

Comment:
Not really related to this PR specifically, but does something clean up these strings once they are no longer used?

Reply:
Good questions. @alexvy86?

Reply:
Not right now. A recent change of mine started keeping track of the mapping from code to original message in a `.ts` file in `@fluidframework/test-runtime-utils` but I didn't change what `policy-check` already did in the line itself (just move the string to a comment). Since we now have the mapping file I guess the comments are technically unnecessary, but I'd find them convenient for when others read the code.

Reply:
Yeah definitely agree the comments are convenient and should stick around